api/src/pcapi/routes/public/documentation_constants/tags.py

```diff
@@ -5,7 +5,10 @@
 BOOKINGS = Tag(name="Bookings", description="Endpoints to manage the bookings of an offer (event and product).")
 
 # ADDRESSES
-ADDRESSES = Tag(name="Addresses", description="[Beta] Endpoints to manage offers addresses")
+ADDRESSES = Tag(
+    name="Addresses",
+    description="Endpoints for managing addresses, allowing you to specify an offer location that differs from the venue's location.",
```

Comment:
(détail) 
Endpoints to manage addresses, allowing you to specify an offer location different from the venue's location.

Reply:
Je crois que pareil c'est la reformulation en `for managing` plutôt que `to manage` vient de chat GPT. En fait les deux formulations sont équivalentes, là il est vrai que je ne respecte par la formulation utilisée pour les autres tags (j'ai un peu la flemme d'attendre une CI complète pour ce changement ^^)